vagrant_support/checksetup_answers.j2

```diff
@@ -0,0 +1,37 @@
+$answer{'ADMIN_EMAIL'}          = 'vagrant@{{ WEB_HOSTNAME }}';
+$answer{'ADMIN_OK'}             = 'Y';
+$answer{'ADMIN_PASSWORD'}       = 'vagrant01!';
```

Comment:
IMHO, This would hinder the flow of the vagrant installation, and would tire the user with technicalities that might not matter to him as such. I am not sure how this should affect the security, until a vagrant installation is used in production. @floatingatoll your opinion on this?